vnext/ReactWindowsCore/tracing/fbsystrace.h

```diff
@@ -51,44 +55,48 @@ class FbSystraceSection {
             .count());
   }
 
-  void init(std::string &&v) {
-    args_[index_++] = std::move(v);
-    begin_section();
+  template <typename... RestArg>
+  FbSystraceSection(uint64_t tag, std::string &&profileName, RestArg &&... rest)
+      : tag_(tag), profile_name_(std::move(profileName)) {
+    id_ = s_id_counter++;
```

Comment:
>id_ [](start = 4, length = 3)

Please move it to the initialization section of the constructor.

Reply:
Actually, I think I'm going to leave this as how Anandraj originally had it. Mutating state in the intialization list feels a bit too subtle.